src/AssetsPage/CopyButton/index.jsx

```diff
@@ -0,0 +1,39 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import Button from '@edx/paragon/src/Button';
+import copy from 'copy-to-clipboard';
+
+export default class CopyButton extends React.Component {
+  constructor(props) {
+    super(props);
+
+    this.state = {
+      wasClicked: false,
+    };
+  }
+
+  render() {
+    return (
+      <Button
+        label={!this.state.wasClicked ? this.props.label : this.props.onCopyLabel}
```

Comment:
The label of the button dynamically switches between the label text and the onCopyLabel text, based on the state of wasClicked. This allows us to display a "Copied!" message (or any other message) inside the actual button, which meets accessibility criteria, allows us not to have to deal with the accessibility issues that accompany a tool-tip, and also affords us more clarity in terms of which URL was copied (compare with a general status alert at the top of the page, which isn't tied to a particular url).

Reply:
Good workaround! The tooltips had me very scared when we were grooming this ticket.